pages/linux/qtile.md

```diff
@@ -0,0 +1,24 @@
+# qtile
+
+> A full-featured, hackable tiling window manager written and configured in Python.
+> More information: <https://docs.qtile.org/en/latest/manual/commands/shell/index.html>.
+
+- Start the window manager, if it is not running already (should ideally be run from `.xsession` or similar):
+
+`qtile start`
+
+- Check a configuration file for any compilation errors (default location is  `~/.config/qtile/config.py`):
+
+`qtile check`
+
+- Show current resource usage information:
+
+`qtile top --force`
```

Comment:
Why does this need `--force`?

Reply:
When run without `--force` (for the first time in a session) the following error message is output

```
tracemalloc not started on qtile, start by setting PYTHONTRACEMALLOC=1 before starting qtile
or force start tracemalloc now, but you'll lose early traces
```

Would this be ok to leave the command as `qtile top`? Not sure what the protocol is here, sorry

Reply:
Ok, that's sensible.